distribution/changelog.txt

```diff
@@ -18,6 +18,7 @@
 - Fix: [#5981] Ride list doesn't update after using quick demolish.
 - Fix: [#5984] Allow socket binding to same port after crash
 - Fix: [#5998] Staff not getting paid / no loan interest.
+- Fix: [#5901] Placing peep spawn not synced across multiplayer
```

Comment:
Needs to be higher up the list. Changelog is order by category, then issue number.